### PR TITLE
fixing menu drawer 

### PR DESCRIPTION
**Pull request by AndreSand**
fixing menu drawer 

Thanks.

### CODE REVIEW

**Comment by dbykovsky** [comment]
@alberto-campos 
Please fix the following bugs first from the previous PR, before pushing a new incomplete feature:
1.There is no default picture but button CAMERA button
2.If you take pic/select existing, picture is not set into image view. You need to leave Profile page and come back again to see profile pic
3.Why is sumofus logo on the right from the Add Credit Card button?
4.Make user be able to update phone number. It's not being saved on parse for now

**Comment by AndreSand** [comment]
fixes for pull 47.

I will fix  4. tomorrow.

@dbykovsky can you merge code? so everyone is on latest code. 

Thanks.

File: app/src/main/java/com/support/android/designlibdemo/activities/MainActivity.java [hunk 1]
```diff
@@ -91,10 +97,29 @@ protected void onCreate(Bundle savedInstanceState) {
         currentUser = ParseUser.getCurrentUser();
         // Skip if already subscribed
         subscribeUserToChannel();
+        //Load image from Parse
+        ParseFile image = (ParseFile) currentUser.getParseFile("profilePicture");
+
+        //Get Image from parse
+        if(image!=null){
```
Comment (dbykovsky) [inline app/src/main/java/com/support/android/designlibdemo/activities/MainActivity.java:104]: this fix is already in master. Please pull the latest code.

File: app/src/main/java/com/support/android/designlibdemo/activities/SignUpActivity.java [hunk 2]
```diff
@@ -1,29 +1,62 @@
 package com.support.android.designlibdemo.activities;
```
Comment (dbykovsky) [inline app/src/main/java/com/support/android/designlibdemo/activities/SignUpActivity.java:1]: We already discussed that 3 times already. We are not planning to enable picture capture functionality for sign up flow.
PLEASE REMOVE THE CODE RELATED TO PICTURE CAPTURING FROM SIGN UP FLOW @alberto-campos

**Comment by dbykovsky** [comment]
This is already in the code. I'm not sure what are you trying do do with
this PR? https://github.com/dbykovsky/sumofusM/pull/4
It looks like you are trying the code that we are already have.

File: app/src/main/java/com/support/android/designlibdemo/activities/SignUpActivity.java (hunk 2 above)
Comment (AndreSand) [inline app/src/main/java/com/support/android/designlibdemo/activities/SignUpActivity.java:1]: I mean: 4
 4.Make user be able to update phone number. It's not being saved on parse for now

**Comment by AndreSand** [comment]
Hi Guys, can you guys merge? I don't see any code conflicts. 

I remove code from signup flow.

I'm pushing UI changes/improvements (not new functionalities) @dbykovsky @alberto-campos 

My fork is up to date with repo M.

Thanks,

**Pull request closed without merging**